front/app/containers/ProjectsShowPage/shared/header/ProjectActionButtons.tsx

```diff
@@ -148,29 +152,41 @@ const ProjectActionButtons = memo<Props>(({ projectId, className }) => {
     project,
     phases
   );
+
   const isParticipationMethodIdeation = participation_method === 'ideation';
+
   const isParticipationMethodNativeSurvey =
-    participation_method === 'native_survey';
+    participation_method === 'native_survey' ||
+    currentPhase?.attributes.participation_method === 'native_survey';
+
   const showSeeIdeasButton =
     ((isProcessTypeContinuous && isParticipationMethodIdeation) ||
       currentPhase?.attributes.participation_method === 'ideation') &&
     isNumber(ideas_count) &&
     ideas_count > 0;
+
   const showIdeasButton =
-    (isParticipationMethodIdeation && publication_status !== 'archived') ||
-    isParticipationMethodNativeSurvey;
+    isParticipationMethodIdeation && publication_status !== 'archived';
```

Comment:
Code like this is an indication that the decision should be delegated to `ParticipationMethodConfig`.

Reply:
OK. Let's do it later.